content/blog/announcing-chapel-1-30-0.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Announcing Chapel 1.30.0!
+date: 2023-03-24T00:29:50.615Z
+priority: ""
```

Comment:
Hi Brad, 

I think I understand the reason for the CMS rendering process failure: the Priority option was probably set to a value then removed.

Using the CMS editor, select your blog link entry, and the could you please set the priority to a value higher than 1. Say 2 or higher value.

Save, then click "Writing in Blog collection / Changes saved" to exit the CMS editor mode and return to the Editorial Workflow. Then let the rendering process to run for a few minutes (4 to 5 minutes).

 This workaround should make the trick.

 I will then merge the PR in github to push the blog post link to the HPE Dev community portal.

Best regards,

Denis

CC: @reddypramod85

Reply:
Thanks Denis — I think you're right that I selected a priority and then attempted to unselect it from the UI.  I've followed your instructions and we'll see how the CI does this time.

Reply:
@choukden : Looks clean this time around—let me know if there's more I need to do here.